week2/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+)
+
+func RemoveSpecialCharacter(text string) string {
+
+	// Make a Regex to say we only want letters and numbers
+	reg, err := regexp.Compile("[^a-zA-Z0-9]+")
+	if err != nil {
+		log.Fatal(err)
+	}
+	validCharacter := reg.ReplaceAllString(text, " ")
+
```

Comment:
You should remove trailing spaces too, you can use [`strings.Fields`](http://golang.org/pkg/strings#Fields) to do it.